pkgs/by-name/holo-cli/package.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  stdenv,
+  rustPlatform,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  protobuf,
+  pcre2,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "holo-cli";
+  version = "0.4.0";
```

[Versioning](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning)

```suggestion
  version = "0.4.0-unstable-2025-03-31";
```